### PR TITLE
Fix/107/backend tests

### PR DESCRIPTION
**Pull request by JamieArmstrongArndt**
Added Slack Notifications and fixed Python Error (Usage of wrong version)

Let us merge this tomorrow together.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +29,16 @@ jobs:
         - npm run test -- --watch=false --no-progress --browsers=ChromeHeadlessNoSandbox
         - echo "no backend / crawler / other tests specified"
 
+    - language: python
+      python: 3.7
+      before_install:
+        - cd scripts/Wikidata\ crawler
+        - python --version
```
Comment (mxxbe) [inline .travis.yml]: we could remove the version print again

File: .travis.yml [hunk 2]
```diff
@@ -25,6 +29,16 @@ jobs:
...
+      install:
+        - pip3 install -r requirements.txt
```
Comment (mxxbe) [inline .travis.yml]: shouldn't it work with just `pip install` now - since we specified the version for the whole job ?

File: .travis.yml (hunk 1 above)
Comment (jschurse) [reply .travis.yml]: Or we leave it. No harm is being done by it and if the tests should fail we can look up the version in the logs.

File: .travis.yml (hunk 2 above)
Comment (jschurse) [reply .travis.yml]: It should, but we can also just leave this more explicit statement.

**Pull request merged**